KinectServer/KinectDevice.cpp

```diff
@@ -8,8 +11,8 @@ KinectDevice::KinectDevice():
 {
     auto validate = [this](int32_t result) {
         this->isValid = result;
-        if (FAILED(this->isValid)) {
-            throw "Initializing Kinect device failed.";
+        if (this->isValid == 0) {
```

Comment:
Fascinating. Do you get a 0 on your machine when running this code? Can we do `<=` instead?